src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -1625,6 +1625,10 @@ private void AddDefaultFiles(string org, string service, string edition)
       File.Copy(_generalSettings.DefaultViewStart, defaultViewFilePath + _settings.DefaultViewStartFileName);
       File.Copy(_generalSettings.DefaultViewImports, defaultViewFilePath + _settings.DefaultViewImportsFileName);
 
+      // Copy default Dockerfile
+      string editionPath = _settings.GetEditionPath(org, service, edition, AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext));
+      File.Copy(_generalSettings.DefaultRepoDockerfile, editionPath + "/" + _settings.DockerfileFileName);
```

Comment:
Looks like the method GetEditionPath returns  $"{RepositoryLocation}{developer}{org}/{service}/{General.EditionsFolder}/{edition}/" so I guess the "/" is unnecessary

Reply:
Fixed